react/features/toolbar/defaultToolbarButtons.js

```diff
@@ -0,0 +1,334 @@
+/* @flow */
+
+import React from 'react';
+
+import UIEvents from '../../../service/UI/UIEvents';
+
+declare var APP: Object;
+declare var config: Object;
+declare var JitsiMeetJS: Object;
+
+/**
+ * Shows SIP number dialog.
+ *
+ * @returns {void}
+ */
+function _showSIPNumberInput() {
+    const defaultNumber = config.defaultSipNumber || '';
+    const titleKey = 'dialog.sipMsg';
+    const msgString = `
+            <input class="input-control"
+                   name="sipNumber" type="text"
+                   value="${defaultNumber}" autofocus>`;
+
+    APP.UI.messageHandler.openTwoButtonDialog({
+        titleKey,
+        msgString,
+        leftButtonKey: 'dialog.Dial',
+
+        // eslint-disable-next-line max-params
+        submitFunction: (event, value, message, formValues) => {
+            const { sipNumber } = formValues;
+
+            if (value && sipNumber) {
+                APP.UI.emitEvent(UIEvents.SIP_DIAL, sipNumber);
+            }
+        },
+        focus: ':input:first'
+    });
+}
+
+/**
+ * All toolbar buttons' descriptions.
+ */
+export default {
+    microphone: {
+        classNames: [ 'button', 'icon-microphone' ],
+        content: 'Mute / Unmute',
+        enabled: true,
+        i18n: 'toolbar.mute',
+        id: 'toolbar_button_mute',
+        onClick() {
+            const sharedVideoManager = APP.UI.getSharedVideoManager();
+
+            if (APP.conference.audioMuted) {
+                // If there's a shared video with the volume "on" and we aren't
+                // the video owner, we warn the user
+                // that currently it's not possible to unmute.
+                if (sharedVideoManager
+                    && sharedVideoManager.isSharedVideoVolumeOn()
+                    && !sharedVideoManager.isSharedVideoOwner()) {
+                    APP.UI.showCustomToolbarPopup(
+                        '#unableToUnmutePopup', true, 5000);
+                } else {
+                    JitsiMeetJS.analytics.sendEvent('toolbar.audio.unmuted');
+                    APP.UI.emitEvent(UIEvents.AUDIO_MUTED, false, true);
+                }
+            } else {
+                JitsiMeetJS.analytics.sendEvent('toolbar.audio.muted');
+                APP.UI.emitEvent(UIEvents.AUDIO_MUTED, true, true);
+            }
+        },
+        popups: [
+            {
+                id: 'micMutedPopup',
+                className: 'loginmenu',
+                dataAttr: 'toolbar.micMutedPopup'
```

Comment:
Sort the object properties in alphabetical order please. i don't see any advantage to this random order. I don't think anyone cares more about `id` than about `className` or `dataAttr`. These are mere collections of key-value pairs.